tests/strategy/test_default_strategy.py

```diff
@@ -32,6 +32,7 @@ def test_strategy_test_v3(result, fee, is_short, side):
     assert type(indicators) is DataFrame
     assert type(strategy.populate_buy_trend(indicators, metadata)) is DataFrame
     assert type(strategy.populate_sell_trend(indicators, metadata)) is DataFrame
+    assert strategy.bot_started is True
```

Comment:
Not sure why this would be failing

Reply:
you can't test this here. It's initializing the strategy directly, independent of both strategy loader, as well as all other bot modules.
As such, it's expected that this is not called here.